app/views/docs/oauth-providers/spotify.md

```diff
@@ -0,0 +1,342 @@
+# Spotify provider
+
+Appwrite allows you to authenticate users using their Spotify account through the Spotify OAuth2 provider. OAuth authentication is a great way to reduce friction for your users and increase user conversion by simplifying the signup process. You can learn more about Appwrite's other OAuth2 providers [here](placeholder link). 
+
+## Enabling the Spotify provider
+Before you can use Spotify to authenticate users, you need to enable the provider in your Appwrite console.
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Spotify OAuth2 Settings** modal, use the toggle to enable the provider
+
+Don't close this modal, we'll need to create a Spotify app to complete this form.
+
+## Creating a Spotify app for OAuth
+To use Spotify OAuth with Appwrite, you need to create a Spotify app using a Spotify developer account. You can do this by following the [Creating Spotify App](https://developer.spotify.com/documentation/web-api/concepts/apps) guide, in the [Spotify Dashboard](https://developer.spotify.com/dashboard). While creating a new Spotify app in the **Redirect URI** filed, provide the **URI** found in the **Spotify OAuth2 Settings** modal from your Appwrite console.
+
+After you've created your Spotify app, you can head back to your Appwrite console to complete the form in the **Spotify OAuth2 Settings** modal.
+- You can find the **Client ID** in your Spotify app settings menu and provide this in the **App ID** field in the **Spotify OAuth2 Settings** modal from the Appwrite console.
+- In the settings menu click on the **View client secret** button to get your apps client secret.
+- Copy the client secret and provide this in the **App Secret** field in the **Spotify OAuth2 Settings** modal from the Appwrite console.
+
+## Authenticating
+You can use any of the Appwrite Client SDKs to authenticate users with their Spotify account.
+
+### Web
+When a user calls the [Create OAuth2 Session](https://appwrite.io/docs/client/account#accountCreateOAuth2Session) endpoint in your web app, they will be taken to Spotify's OAuth page to complete their login. 
+
+After authenticating, they'll be redirected back to your app using either the `success` or `failure` URLs provided. To provide the best experience to your users, make sure to **implement and provide both routes** to prompt the user about successful and failed authentication attempts.
+
+```js
+import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('[PROJECT_ID]') // Your project ID
```

Comment:
I know this isn't your fault, I think my examples are wrong, too, but:

```
client
    .setEndpoint('https://cloud.appwrite.io/v1')  // Your API Endpoint
    .setProject('[PROJECT_ID]')                          // Your project ID
```

See if you can align the comments.

Reply:
Sure! I will do that 😄